tests/host/common/Arduino.cpp

```diff
@@ -81,3 +81,4 @@ extern "C" void cont_yield(cont_t*)
 {
 }
 
+const char* overrideme PROGMEM = " should be overridden for better efficiency\r\n";
```

Comment:
I think you placed the char pointer in flash, but left the string in RAM.  Suggest
`const char overrideme [] PROGMEM = " should be overridden for better efficiency\r\n";`

Reply:
This is a leftover, but good eyes anyway!